client/app/cda/details/cda-details.controller.js

```diff
@@ -13,11 +13,9 @@ angular.module("managerApp")
         self.guides = {
             title: $translate.instant("cda_guide_title"),
             list: [{
-                name: $translate.instant("cda_guide_name"),
-                url: ovhDocUrl.getDocUrl("cloud/storage/ceph")
```

Comment:
This uses 2API. I've removed altogether as we're not supposed to use 2API if we can prevent it.